### DISCUSSION TITLE
请问为什么gpu运行不了

### DISCUSSION THREAD

**Discussion by Drewhhh** [Comfy-Org/ComfyUI#2343] [Q&A]
![Snipaste_2023-12-21_22-30-23](https://github.com/comfyanonymous/ComfyUI/assets/148340483/c6b9cef1-ffdb-4363-ae6e-540e1a3270b4)
请问为什么GPU模式运行不了 电脑是4G显存的 而且用了--lowvarm也不行   但是cpu可以    求大神解答

**Comment by KerfuffleV2** [comment]
@Drewhhh 错误信息说你的GPU driver太旧了，需要你安装新的，还有另一个办法：安装包括比较新的GPU driver的PyTorch版本，这样也有可能会解决了问题。我只是个汉语的学生，请原谅我写得不怎么流利吧！

**Comment by Drewhhh** [comment]
Thank you.&nbsp;
I’ve successfully solved my problem.



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年12月24日(周日) 凌晨2:43
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [comfyanonymous/ComfyUI] 请问为什么gpu运行不了 (Discussion #2343)




 
@Drewhhh 错误信息说你的GPU driver太旧了，需要你安装新的，还有另一个办法：安装包括比较新的GPU driver的PyTorch版本，这样也有可能会解决了问题。我只是个汉语的学生，请原谅我写得不怎么流利吧！

**Reply by hzx671779481** [reply]
怎么解决的，我的问题和你一样

**Reply by Drewhhh** [reply]
升级一下显卡驱动
